README.md

```diff
@@ -1,33 +1,33 @@
 # Arrayzy
 
-A native PHP arrays easy manipulation library in OOP way.
+A native PHP library for manipulating arrays in an OOP way.
```

Comment:
I like it. I'd untangle it slightly though, how about this:

    The wrapper for all PHP built-in array functions and easy, object-oriented array manipulation library. In short: Arrays on steroids.